client_overlay/bin/x64/res/qml/MyToggleButton.qml

```diff
@@ -7,24 +7,24 @@ CheckBox {
 	hoverEnabled: true
 	spacing: 12
 
-	indicator: Rectangle {
-		implicitWidth: 28
-		implicitHeight: 28
-		x: parent.leftPadding
-		y: parent.height / 2 - height / 2
-		color: parent.activeFocus ? myPalette.mid : myPalette.base
-		border.width: 0
-		Image {
-			width: 38
-			height: 38
-			x: (parent.width - width) / 2
-			y: (parent.height - height) / 2
-			source: "image://default/check/#ffffff"
```

Comment:
These elements come from the initial parent project https://github.com/matzman666/OpenVR-InputEmulator 
I have had to make a few adjustments with newer QT versions etc, but it mostly works on Windows pretty well, Linux seems to have a few more conflicts with some of the QT elements